### PR TITLE
Change cylinder defalut rotation

### PR DESCRIPTION
**Pull request by p1003**

### CODE REVIEW

File: converter/shieldhit/parser.py [hunk 1]
```diff
@@ -220,8 +220,8 @@ def _get_material_id(self, uuid: str) -> int:
         """Find material by uuid and retun its id."""
         for idx, item in enumerate(self.geo_mat_config.materials):
             if item[0] == uuid:
-                # If the material is a DEFALUT_MATERIAL then we need the value not its index
-                if item[1] in list(DEFALUT_MATERIALS.values()):
+                # If the material is a DefaultMaterial then we need the value not its index
+                if item[1] in DefaultMaterial._value2member_map_:
```
Comment (grzanka) [inline converter/shieldhit/parser.py:224]: `_value2member_map_` is undocumented Python feature. Add a comment here that you are looking for an enum entry with given value.

Comment (retkiewi) [reply converter/shieldhit/parser.py:226]: I've added the comment. Could you change deepsource settings so this doesn't block the PR? I don't have the required rights.

Comment (grzanka) [reply converter/shieldhit/parser.py:226]: This should silence deepsource:
```suggestion
                if item[1] in DefaultMaterial._value2member_map_:  # skipcq: PYL-W0212
```

File: converter/shieldhit/geo.py [hunk 2]
```diff
@@ -252,5 +262,5 @@ def get_mat_string(self) -> str:
         material_strings = [
             self.material_template.format(idx=idx+1, mat=mat[1])
             for idx, mat in enumerate(self.materials)
-            if mat[1] not in list(DEFALUT_MATERIALS.values())]
+            if mat[1] not in DefaultMaterial._value2member_map_]
```
Comment (grzanka) [inline converter/shieldhit/geo.py:265]:
```suggestion
            if mat[1] not in DefaultMaterial._value2member_map_]  # skipcq: PYL-W0212
```

File: converter/shieldhit/geo.py [hunk 3]
```diff
@@ -1,8 +1,19 @@
...
 from dataclasses import dataclass, field
 from math import log10, ceil, isclose
 from scipy.spatial.transform import Rotation as R
+from enum import IntEnum
+
+
+class DefaultMaterial(IntEnum):
+    """
+    List of special material ids that are not refering to the `mat` and have a separete
+    meaining. E.g. Asigning id=1000 to a zone in `geo` file would mean, that the zone is made out
```
Comment (grzanka) [inline converter/shieldhit/geo.py:11]:
```suggestion
    meaning. E.g. Assigning id=1000 to a zone in `geo.dat` file would mean, that the zone is made out
```

File: converter/shieldhit/geo.py [hunk 4]
```diff
@@ -1,8 +1,19 @@
...
+    of vacuum, not that it refers to the 1000th item in `mat` file.
```
Comment (grzanka) [inline converter/shieldhit/geo.py:12]:
```suggestion
    of vacuum, not that it refers to the 1000th item in `mat.dat` file.
```

File: converter/shieldhit/geo.py [hunk 5]
```diff
@@ -1,8 +1,19 @@
...
-from converter.shieldhit import DEFALUT_MATERIALS
 from dataclasses import dataclass, field
 from math import log10, ceil, isclose
 from scipy.spatial.transform import Rotation as R
+from enum import IntEnum
+
+
+class DefaultMaterial(IntEnum):
+    """
+    List of special material ids that are not refering to the `mat` and have a separete
```
Comment (grzanka) [inline converter/shieldhit/geo.py:10]:
```suggestion
    List of special material ids that are not referring to the `mat.dat` and have a separate
```

**Pull request closed without merging**